nxsdk_modules_contrib/temporal_diff_enc/stde.py

```diff
@@ -0,0 +1,238 @@
+# Copyright(c) 2020 University of Zurich. All rights reserved.
```

Comment:
Please remove the copyright and license information from all files. We include a LICENSE https://github.com/intel-nrc-ecosystem/models/blob/master/LICENSE.

Instead please modify the README https://github.com/intel-nrc-ecosystem/models/blob/master/README.md Acknowledgement Section with your network module and authorship. You may add any other third party acknowledgements and citations in a local README in your module.